Gemfile.lock

```diff
@@ -192,19 +192,16 @@ DEPENDENCIES
   figaro
   jbuilder (~> 2.5)
   listen (>= 3.0.5, < 3.2)
-  pg
```

Comment:
This needs to be `bundle install` again because there should be no changes to this file